CONTRIBUTING.rst

```diff
@@ -0,0 +1,76 @@
+Contributor Guide
+=================
+
+* Thank you for participating!
+* A contribution(pull request) requires to have one `issue <https://github.com/rucio/rucio/issues/new>`_ created.
```

Comment:
I think it'll happen to get the pull request before the issue..

Reply:
Well, that can be, yes, but we should seriously discourage it. We want to avoid the situation where people submit things that have not been at least mentioned before.

Reply:
this is how things can happen on github ;). pull requests + extensive commit message are one way to get contribution.but here the answer should be please (or someone else can do) create an issue with the description etc and then possible discussion

Reply:
I still disagree that the discussion should only come after the code. Otherwise we can run in situations where people spend a lot of time developing something in their corner that might not be merged later on. This should be super clear from the beginning and can be easily avoided by having the ticket first.

Reply:
I don't think you get my point which was based on my experience. What you said is true among the core dev/collaborator with meeting etc. but on github you get ideally contributions from more places, others experiments, etc. (cf. this morning discussion). I actually see direct pul request for dcache, etc.

Reply:
Yes, but even there if somebody wants to develop something, it's not asked too much to create an issue beforehand, no? It could be that somebody is already working on it for example. I don't think the phrasing is to extreme, we just encourage to create an issue early, but off course it will happen from time to time that the issue only comes with the pull request but that doesn't mean we have to encourage it.